modules/debts_api/lib/debts_api/v0/financial_status_report_service.rb

```diff
@@ -142,8 +146,11 @@ def submit_vha_fsr(form_submission)
     def submit_to_vbs(form_submission)
       form = add_vha_specific_data(form_submission)
 
+      request_start_time = Time.current
       vbs_request = DebtManagementCenter::VBS::Request.build
       Rails.logger.info('5655 Form Submitting to VBS API', submission_id: form_submission.id)
+      StatsD.measure("#{STATSD_KEY_PREFIX}.fsr.submit.vbs.latency", (Time.current - request_start_time) * 1000)
```

Comment:
this one here needs to go below the code underneath it on 154. That's the request there.

Reply:
`vbs_request.post("#{vbs_settings.base_path}/UploadFSRJsonDocument",
                       { jsonDocument: form.to_json })`

This stuff ☝🏻

Reply:
might even want to pull that request_start_time down just above the reqquest